src/map/packets/char_update.cpp

```diff
@@ -103,6 +103,44 @@ CCharUpdatePacket::CCharUpdatePacket(CCharEntity* PChar)
     }
     ref<uint64>(0x4C) = PChar->StatusEffectContainer->m_Flags;
 
+    //----------------------------------------------------------------------------------
+    // GEO bubble effects, changes bubble type depending on what effect is activated.
+    //----------------------------------------------------------------------------------
+    if (PChar->StatusEffectContainer->HasStatusEffect(EFFECT_INDI_POISON))
+	    ref<uint8>(0x58) = 0x5D;
```

Comment:
Please wrap up these conditions with braces:
```
if (condition)
{
... stuff
}
```

Reply:
This id gonna be changed in the next push i do and will get rid of all this.

Reply:
>  will get rid of all this.

:+1: 
was gonna ask why you were checking the status instead of what spell cast it rode in on.